system/Debug/Exceptions.php

```diff
@@ -80,6 +80,8 @@ public function __construct(ExceptionsConfig $config, $request, ResponseInterfac
         $this->response = $response;
 
         // workaround for upgraded users
+        // This causes "Deprecated: Creation of dynamic property" in PHP 8.2.
+        // @TODO remove this after dropping PHP 8.1 support.
```

Comment:
Why this todo?

Reply:
Just a reminder. When we support only PHP 8.2+, the Config file must have the properties.